README.md

```diff
@@ -207,3 +207,16 @@ After you create a [Sauce Labs](https://saucelabs.com) account:
 - Unit tests do not require running the dummy API.
 - To run the unit tests along with the functional tests: ```grunt test``` from the root of the repo.
 - To run unit tests individually: ```grunt mocha_istanbul``` from the root of the repo.
+
+### Font files
+
+The CSS styles for this project refer to some font files which cannot be distributed publicly (see definitions in `regulations/static/regulations/css/less/fonts.less`). These fonts need to live under a `fonts` subdirectory of wherever static assets are served from, e.g. `/static/fonts`. There are fallback fonts defined which will be used if the desired fonts are not available.
+
+If you want to install self-hosted fonts locally for use in development, you can place the font files in `/static.in/cfgov-fonts/fonts/` and restart the local web server. If you are a CFPB employee, you can perform this step with:
+
+```
+cd /static.in/ && git clone https://[GHE]/CFGOV/cfgov-fonts/
```

Comment:
This might just be me, but I had to drop the leading `/` in `/static.in/` to switch to the right directory (I was in the root of `regulations-site` at the time):

```
cd static.in/ && git clone https://[GHE]/CFGOV/cfgov-fonts/
```

Reply:
Yes, you're right, it should be without a leading `/`. Change made in 35970c5.